.ci/bl-specific.sh

```diff
@@ -1,7 +1,4 @@
 #!/bin/bash
 
 # Perform beamline-specific actions in this file.
-
-sudo mkdir -v -p /GPFS/CENTRAL/xf17id1/skinnerProjectsBackup/bnlpx_config/fmx_bluesky_config/
-sudo chown -Rv $USER: /GPFS/CENTRAL/xf17id1/skinnerProjectsBackup/bnlpx_config/fmx_bluesky_config/
-pip install git+https://github.com/NSLS-II-AMX/mxtools.git
+sudo mkdir -p /nsls2/data/fmx/shared/config/fmx_bluesky_config/
```

Comment:
Lower-cased `fmx` could have been parametrized in this line using the already defined [`BEAMLINE_ACRONYM`](https://github.com/NSLS-II/profile-collection-ci/blob/d6d232479f8c48552bc36d2f9dad0db844a61bc9/2022-2.3-py39.yml#L11-L12) env var (that is upper-case), which could be easily changed with `${BEAMLINE_ACRONYM,,}` (credits to https://stackoverflow.com/a/2265268 - just learned about this option).